lib/Basics/process-utils.cpp

```diff
@@ -86,6 +88,13 @@
 #include "Logger/Logger.h"
 #include "Logger/LoggerStream.h"
 
+#ifdef __APPLE__
+// The following hack is required to get access to the environment variables
+// in the same way as on Linux:
+#include <crt_externs.h>
+#define environ (*_NSGetEnviron())
+#endif
```

Comment:
Technically not necessary, as we do not support building on macOS anymore.

Reply:
Will remove. This is an artifact from the 3.11 forward port.